framework/include/bcs/ADMatchedValueBC.h

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ADNodalBC.h"
+
+class ADMatchedValueBC;
+
+template <>
+InputParameters validParams<ADMatchedValueBC>();
```

Comment:
Can remove

Reply:
Ok. I found this in the original file, so I thought we were still doing that.